db/migrate/20230301071240_add_auto_bump_cooldown_to_category_settings.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class AddAutoBumpCooldownToCategorySettings < ActiveRecord::Migration[7.0]
+  def change
+    add_column :category_settings, :auto_bump_cooldown, :integer, default: 1
```

Comment:
```suggestion
    add_column :category_settings, :auto_bump_cooldown_days, :integer, default: 1
```

What do you think about including the units for the column in the column name itself? It'll make it alot clearer what the unit of the column is without having to read through the code to figure this out.

Reply:
I think the ideal solution would actually be to use the `ActiveRecord::Types` API to serialize and deserialize durations, as the unit is actually by right part of the data, not identifiers, but also didn't want to extend the scope too far. 🙂  We can add `days` for now.